website/pages/docs/agent/options.mdx

```diff
@@ -808,6 +808,53 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
 
 - `advertise_addr_wan` Equivalent to the [`-advertise-wan` command-line flag](#_advertise-wan).
 
+- `audit` <EnterpriseAlert inline /> - Added in Consul 1.8, the audit object allow users to enable auditing
+    and configure a sink and filters for their audit logs.
+
+    ```
+      audit {
+        enabled = true
+        sink "My sink" {
+          type   = "file"
+          format = "json"
+          path   = "data/audit/audit.json"
+          delivery_guarantee = "best-effort"
+          rotate_duration = "24h"
+          rotate_max_files = 15
+          rotate_bytes = 25165824
+        }
+    }
+    ```
+
+    The following sub-keys are available:
+
+    - `enable` - Controls whether Consul logs out each time a user
+    performs an operation. ACLs must be enabled to use this feature. Defaults to `false`.
+
+    - `sink` - This object provides users configuration for where Consul
+    logs auditing events to. Sink is an object containing keys to sink objects, where the key is the name of the sink. 
```

Comment:
should this be `for where Consul auditing events go`?

Reply:
Reworded to remove dangling participle.